src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -64,6 +64,11 @@
             <tag name="sylius.api.command_data_transformer" />
         </service>
 
+        <service id="Sylius\Bundle\ApiBundle\DataTransformer\LoggedInCustomerAwareContactCommandDataTransformer">
```

Comment:
Couldn't you use `LoggedInCustomerEmailAwareCommandDataTransformer` instead of creating the new one?

Reply:
I'm marking it as resolved, as after discussion we need it because of different behavior